pkgs/tools/security/swtpm/default.nix

```diff
@@ -21,17 +24,17 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-7YzdwGAGECj7PhaCOf/dLSILPXqtbylCkN79vuFBw5Y=";
   };
 
-  patches = [
-    (fetchpatch {
-      url = "https://patch-diff.githubusercontent.com/raw/stefanberger/swtpm/pull/527.patch";
-      sha256 = "sha256-cpKHP15a27ifmmswSgHoNzGPO6TY/ZuJIfM5xLOlqlU=";
-    })
-  ];
+  postPatch = ''
+    patchShebangs tests/*
+  '';
 
   nativeBuildInputs = [
     pkg-config unixtools.netstat expect socat
     perl # for pod2man
     autoreconfHook
+
+    # tests
+    python3 which
```

Comment:
test-only dependencies should go to `checkInputs`

Reply:
upstream's configure checks for it (for tests), should we cross compile even if we did not run the tests, we still need to carry python.

Reply:
still doesn't build with `doCheck = false`